### ISSUE TITLE
Replace usage of semicolons by "&&"

### ISSUE THREAD

**Issue by umed** [OpenVisualCloud/Dockerfiles#18]
We need to avoid usage of semicolons.
In case one command finished with failure the `RUN` won't be marked as failed it just execute next command after semicolon.

For example:

```dockerfile

RUN git clone -b ${DLDT_VER} ${DLDT_REPO}; \
    cd dldt; \
    git submodule init; \
    git submodule update --recursive; \
    cd inference-engine; \
    wget -O - ${DLDT_C_API_REPO} | patch -p2; \
```

If `git clone` in the above command will fail this `RUN` won't fail  and you may not even notice that something goes wrong, it just execute next commands.

**Comment by umed** [comment]
Related issue #26

**Comment by dpatel257** [comment]
Thanks for raising this. Been working on to fix this, seeing overlooked issues.

**Comment by dpatel257** [comment]
Merged in master with #40
